packit/api.py

```diff
@@ -234,12 +240,13 @@ def sync_from_downstream(
         self.up.create_branch(local_pr_branch)
         self.up.checkout_branch(local_pr_branch)
 
-        sync_files(
-            self.package_config.synced_files.raw_files_to_sync,
-            self.dg.local_project.working_dir,
-            self.up.local_project.working_dir,
+        raw_sync_files = self.package_config.synced_files.get_raw_files_to_sync(
```

Comment:
> Why separate it in two methods when they are always used together? 

Testing...

Reply:
I did not write the original code, I just took the existing code and made it work for the "sync from downstream" service job.

I agree with you that this can be improved even further. So let's create an issue out of your comment to track the refactoring request.

Reply:
> And also - you set SyncFilesItem from config, only to transfer it to RawSyncFilesItem later - why adding additional layer?

- `SyncFilesItem` can contain wildcards and multiple sources.
- `RawSyncFilesItem` has only one source and it cannot contain wildcards and wildcard expansion needs to be done in the real directory. Not in the time, when `SyncFilesItem` is loaded from the config file.